### PR TITLE
fix : Filterlist null if no filter selected

### LINKED ISSUE #985: Account list is null if no filter is selected
**Summary:**
When the user opens filter options, and if he/she doesn't select any options and clicks filter, then the list is null 
**Steps to reproduce:**
1. Open accounts
2. Open filters, don't select any option and click filter
3. We see that the list disappears.

**Expected behaviour:**
If no filter is selected, all the accounts should be displayed.

**Observed behaviour:**

Blank page

### PR DESCRIPTION
**Pull request by shiv07tiwari**
Fixes #985 

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by shiv07tiwari** [comment]
videotogif_2019 01 22_15 46 46

**Comment by shiv07tiwari** [comment]
videotogif_2019 01 22_21 01 05

**Comment by shiv07tiwari** [comment]
@luckyman20 Made the changes.

File: app/src/main/java/org/mifos/mobilebanking/ui/adapters/CheckBoxAdapter.java [hunk 1]
```diff
@@ -94,4 +123,15 @@ public void checkChanges() {
             statusList.get(getAdapterPosition()).setChecked(cbStatusSelect.isChecked());
         }
     }
+
+    public void setDialogBuilder(MaterialDialog.Builder dialogBuilder) {
+        this.dialogBuilder = dialogBuilder;
+    }
+    void checkIfShouldBeEnabled() {
```
Comment (luckyman20) [inline app/src/main/java/org/mifos/mobilebanking/ui/adapters/CheckBoxAdapter.java]: Add comment for this function.

**Comment by shiv07tiwari** [comment]
@luckyman20 Added the comments.

File: app/src/main/java/org/mifos/mobilebanking/ui/adapters/CheckBoxAdapter.java (hunk 1 above)
Comment (luckyman20) [reply app/src/main/java/org/mifos/mobilebanking/ui/adapters/CheckBoxAdapter.java]: Rename this to toogleStatus

**Comment by shiv07tiwari** [comment]
@miPlodder @therajanmaurya Kindly review this PR.

**Comment by shiv07tiwari** [comment]
@therajanmaurya Fixed the merge conflicts.

**Comment by geekanamika** [comment]
Solved in #1200 
I apologize that I overlooked the old PR.

**Pull request closed without merging**